### ISSUE TITLE
switch stream between 'explore' and 'my network'

### ISSUE THREAD

**Issue by gyst** [ploneintranet/ploneintranet#793]
Allow 'explore' in addition to the networked filtered stream view.

**Comment by gyst** [comment]
On 13/11/15 13:37, Alexander Pilz wrote:

> I was under the impression that a user only sees posts on the
> dashboard from his workspaces or people he follows. We have tried it
> out and users see everything that is posted on the dashboard. Thats
> not an issue but throws up the question what we need the following
> for.

That is a bug. The intent is to show only updates from your network. That's hardcoded here:

https://github.com/ploneintranet/ploneintranet/blob/master/src/ploneintranet/layout/browser/templates/dashboard.pt#L38

That setting is propagated here:
https://github.com/ploneintranet/ploneintranet/blob/master/src/ploneintranet/layout/browser/templates/dashboard.pt#L38

But it's not honored here:
https://github.com/ploneintranet/ploneintranet/blob/master/src/ploneintranet/activitystream/browser/tiles/stream.py#L104
- it should ask for `user_values` if `explore==True` (fetch the userlist from ploneintranet.network and filter the updates on that userlist).

This feature used to work in plonesocial and the backend should still there and operational.

> Do you have an answer for me for the ikath people?

We have an open ticket to enable switching between 'my network' and 'explore' - both are already implemented in the prototype and in the backend.
https://github.com/ploneintranet/ploneintranet/issues/793

I will attach this message to that ticket.

So likely that ticket should be rephrased to enable 'my network' (since 'explore' is already there instead of the other way around). Actually it's good that 'explore' is the default, avoids an empty stream. But we should add the option to filter on 'my network'.

Implementing this should take less time than I took to write this mail. Which goes to show that it always takes longer than you think :-(. I'm best positioned to implement this easily since I wrote the code in the first place.
